frontend/packages/shared/src/components/FormField/FormField.tsx

```diff
@@ -102,28 +103,18 @@ export const FormField = <T extends unknown, TT extends unknown>({
     if (!errCode && onChange) onChange(newValue, event, errorCode);
   };
 
-  const renderChildren = (childList: React.ReactNode) => {
-    return React.Children.map(childList, (child) => {
-      if (React.isValidElement(child)) {
-        const props =
-          typeof child.type !== 'string'
-            ? {
-                value: tmpValue,
-                required: isRequired,
-                label,
-                onChange: handleOnChange,
-                ...child.props,
-              }
-            : {};
-
-        if (errorCode) {
-          props['aria-errormessage'] = errorMessageId;
-          props['aria-invalid'] = true;
-        }
-
-        return React.cloneElement(child, props);
-      }
-    });
+  const generateProps = () => {
+    const props: FormFieldChildProps<TT> = {
+      value: tmpValue,
+      required: isRequired,
```

Comment:
Jeg ser nå at vi må kalle denne `customRequired` for at det skal stemme overens med resten av koden.
```suggestion
      customRequired: isRequired,
```

Reply:
endret